physics/module_SGSCloud_RadPre.meta

```diff
@@ -298,6 +316,38 @@
   type = logical
   intent = in
   optional = F
+[imp_physics]
+  standard_name = flag_for_microphysics_scheme
+  long_name = choice of microphysics scheme
+  units = flag
+  dimensions = ()
+  type = integer
+  intent = in
+  optional = F
+[imp_physics_gfdl]
+  standard_name = flag_for_gfdl_microphysics_scheme
+  long_name = choice of GFDL microphysics scheme
+  units = flag
+  dimensions = ()
+  type = integer
+  intent = in
+  optional = F
+[imp_physics_thompson]
```

Comment:
Are imp_physics_thompson and imp_physics_wsm6 used in your implementation? I see you added them to the argument list, but the clunky GitHub diff view tells me that neither of them is used in the code.

Reply:
These options aren't used yet, but I can see places where we may want to add options in the future since Thompson has distinct bias characteristics that we may need to improve upon. I just added wsm6 for our NCAR colleagues, since they may want to add other options in the future for their suite.

Reply:
I would remove them for now.